esp/platform/espcfg.cpp

```diff
@@ -144,6 +213,14 @@ void CEspConfig::ensureSDSSessionDomains()
             throw MakeStringException(-1, "Failed to connect to %s.", PathSessionRoot);
 
         ensureESPSessionInTree(conn->queryRoot(), m_process.str());
+
+        if (serverSessionTimeoutSeconds > ESP_SESSION_NEVER_TIMEOUT)
```

Comment:
I think "serverSessionTimeoutSeconds != ESP_SESSION_NEVER_TIMEOUT" is safer than presuming ESP_SESSION_NEVER_TIMEOUT is a negative number